campaignion_email_to_target/src/Wizard/TargetStep.php

```diff
@@ -70,7 +70,7 @@ public function stepForm($form, &$form_state) {
       ],
       [
         'key' => 'salutation',
-        'description' => 'This field is how the target will be addressed in the email opening so should include any appropriate honorifics e.g. ‘Rt Hon John Smith MP’',
+        'description' => 'This field is how the target will be addressed in the message opening so should include any appropriate honorifics e.g. ‘Rt Hon John Smith MP’',
```

Comment:
We will need to actually change these labels in [e2t-data](https://gitlab.more-onion.com/e2t/e2t-data) too. Otherwise they will still show up differently in the token info.

Reply:
I looked through the dataset.yaml files and the descriptions differ. But I think it makes sense that it does.
e2t-data has a more generic approach, for instance:
Salutation = Spoken form of personal address; also used at start of letter.
while the Drupal code features a more help text style user centred description. Do you think they should be the same?